### PR TITLE
Ignore current record when checking for existing username

### PR DESCRIPTION
**Pull request by fritzmg**
Follow up to #21 - otherwise you won't be able to update existing records in the back end anymore.

### CODE REVIEW

File: src/EventListener/MailUsernameListener.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ public function saveMemberEmail($strValue, $dc)
             $this->connection->executeQuery('LOCK TABLES tl_member WRITE');
 
             // Check if the username already exists
-            $exists = $this->connection->fetchOne('SELECT TRUE FROM tl_member WHERE username = ?', [$strValue]);
+            $exists = $this->connection->fetchOne('SELECT TRUE FROM tl_member WHERE username = ? AND id != ?', [$strValue, $dc->id]);
```
Comment (ameotoko) [inline src/EventListener/MailUsernameListener.php:65]: Doesn't it produce a warning in frontend with PHP 8? (see #25)

Comment (ameotoko) [inline src/EventListener/MailUsernameListener.php:65]: Doesn't it produce a warning in frontend with PHP 8? (see https://github.com/terminal42/contao-mailusername/issues/25)

Comment (fritzmg) [reply src/EventListener/MailUsernameListener.php:65]: I suppose so - I'l check again today.

**Comment by aschempp** [comment]
fixed in bf5789c9e5fdd3084a8c31502f91da09d45fe78c

**Pull request closed without merging**